bindings/python/examples/hello_module.py

```diff
@@ -0,0 +1,50 @@
+import coreir
```

Comment:
We moved our pycoreir out of coreir repo a while ago. Can you remoe this file or add it to Lenny's pycoreir?